src/System.Linq.Expressions/tests/Unary/UnaryQuoteTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+using static System.Linq.Expressions.Expression;
+
+namespace System.Linq.Expressions.Tests
+{
+    public class UnaryQuoteTests
```

Comment:
More tests for `Quote` and `RuntimeOps` will come separately as part of https://github.com/dotnet/corefx/issues/11409.